### PR TITLE
docs: fix example code to get token in data-api

### PR DESCRIPTION
**Pull request by rafaelbmateus**
this PR fix documentation about get token in data-api
and small change in response status

### CODE REVIEW

**Review by caibar — APPROVED** [review]
Nice!

**Review by caibar — APPROVED** [review]

**Review by caibar — APPROVED** [review]

File: source/includes/_data.en.md [hunk 1]
```diff
@@ -11,28 +11,20 @@ package main
...
-  _ = writer.WriteField("grant_type", "client_credentials")
-  err := writer.Close()
-  if err != nil {
-    fmt.Println(err)
-    return
-  }
-
+  payload := strings.NewReader(`{ 
+   "application": "rafael-mateus-app",
+   "application_secret": "uMPJMHGsUx"
```
Comment (lborguetti) [inline source/includes/_data.en.md]: @rafaelbmateus could you change this application_secret to `<your-secret>`?

**Review by lborguetti — APPROVED** [review]
LGTM!

File: source/includes/_data.en.md (hunk 1 above)
Comment (lborguetti) [reply source/includes/_data.en.md]: @rafaelbmateus thanks =]

Comment (rafaelbmateus) [reply source/includes/_data.en.md]: My fault, fixed in https://github.com/NeowayLabs/docs-api/pull/8/commits/31902cc2b226b116b255326dc7004f1bc0c090b4

**Pull request merged**